addons/l10n_lt/data/account.account.template.csv

```diff
@@ -0,0 +1,184 @@
+"id","name","code","user_type_id/id","chart_template_id/id","tag_ids/id","reconcile"
+"account_account_template_1130","Software Acquisition Cost",1130,"account.data_account_type_non_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5","False"
+"account_account_template_1138","Amortization of Software Value (-)",1138,"account.data_account_type_non_current_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_5","False"
+"account_account_template_1200","Land Acquisition Cost",1200,"account.data_account_type_fixed_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_10","False"
+"account_account_template_1201","Change of Land Value after Revaluation",1201,"account.data_account_type_fixed_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_10","False"
+"account_account_template_1210","Acquisition Cost of Buildings",1210,"account.data_account_type_fixed_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_11","False"
+"account_account_template_1212","Buildings being Prepared for Use",1212,"account.data_account_type_fixed_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_11","False"
+"account_account_template_1217","Depreciation of the Acquisition Cost of Buildings (-)",1217,"account.data_account_type_fixed_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_11","False"
+"account_account_template_1220","Acquisition Cost of Machinery and Equipment",1220,"account.data_account_type_fixed_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_12","False"
+"account_account_template_1222","Machinery and Equipment being Prepared for Use",1222,"account.data_account_type_fixed_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_12","False"
+"account_account_template_1227","Depreciation of the Acquisition Cost of Machinery and Equipment (-)",1227,"account.data_account_type_fixed_assets","l10n_lt.account_chart_template_lithuania","l10n_lt.account_account_tag_12","False"
```

Comment:
Not sure what the (-) means

Reply:
It is used to emphasize debts/depreciation/loss etc to the company. 
It is actually used by official LT COA sample. You can see it in reference link in PR description: https://e-seimas.lrs.lt/portal/legalAct/lt/TAD/TAIS.225698?jfwid=-pw6827o3s

It is in LT language, but there are same `(-)` for same type of accounts as here.

Reply:
ok, cool

Reply:
@oerp-odoo Referenced link shows already invalidated document since 2016. Here is the current one: https://e-seimas.lrs.lt/portal/legalAct/lt/TAD/665393c0089911e5b0d3e1beb7dd5516

The `(-)` is still there, so that's ok.

Reply:
Thanks, somehow didnt find latest one.